tsl/test/isolation/expected/cagg_insert.out

```diff
@@ -1,141 +1,199 @@
-Parsed test spec with 12 sessions
+Parsed test spec with 17 sessions
 
-starting permutation: LockInvalThrEx Refresh2 Refresh UnlockInvalThrEx
+starting permutation: invalidation_threshold_insert_wait invalidation_threshold_cagg_1 invalidation_threshold_cagg_2 invalidation_threshold_insert_release
+step invalidation_threshold_insert_wait: SELECT debug_waitpoint_enable('invalidation_threshold_insert');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step invalidation_threshold_cagg_1: 
+  CREATE MATERIALIZED VIEW cagg_1
+  WITH (timescaledb.continuous) AS
+  SELECT time_bucket('5', time), COUNT(location)
+  FROM ts_continuous_test_1
+  GROUP BY 1
+  WITH NO DATA;
+
```

Comment:
And it is correct, no??? The other session should wait for the debug point, no?

Reply:
I might be wrong, but I expected both sessions should block on the waitpoint to create the race condition, like this:

![DebugWaitpointCorrect](https://github.com/timescale/timescaledb/assets/1175692/db8d4666-4eb7-4955-91ad-6e9ac116fd0a)

I think this is what is happening right now:

![DebugWaitpointBad](https://github.com/timescale/timescaledb/assets/1175692/e447f125-99d1-4663-a4ed-bab50658fba2)

Reply:
You're correct @mkindahl creating the trigger on the original hypertable requires an AccessExclusiveLock on the relation, so the waiting is because this step and not because the waitpoint.